### ISSUE TITLE
Is this suppose to happen? 

### ISSUE THREAD

**Issue by mug4code** [glisco03/isometric-renders#95]
https://github.com/gliscowo/isometric-renders/assets/67157632/08f8557e-da80-476a-b841-b2805623379e

the individual blocks work, but trying to render an area makes this

**Comment by gliscowo** [comment]
No, this is certainly not supposed to work this way - some error is clearly occurring when it tries to build the area mesh. Can you send me you `latest.log` file? (Check https://fabricmc.net/wiki/player:tutorials:logs if you don't know how to find it)

Cheers

**Comment by mug4code** [comment]
latest.log
<https://drive.google.com/file/d/1QCosFzxs8K3WepzP0ZIWfCZy2FT9HULo/view?usp=drive_web>
I  replied over email as the log file was too large

**Comment by mug4code** [comment]
can you confirm that you have received it?

**Comment by gliscowo** [comment]
I cannot, in fact, confirm that I have received it since it appears you did not make the file available to everybody with the link. I'd personally also be fine if you simply uploaded it here on GitHub as an attachment to a comment

Cheers
